src/lib.rs

```diff
@@ -183,3 +183,17 @@ fn clockpoly(
     polygon
 }
 
+#[cfg(test)]
+mod tests {
+    use super::*;
+
+    #[test]
+    fn internal() {
+        let gj = clockboard();
```

Comment:
Ah, when I tried this out, I got a compiler error:

```
error[E0283]: type annotations needed
    --> src/lib.rs:192:51
     |
192  |         let gj = clockboard(Point::new(0.0, 0.0), Params::from_iter(Vec::new()));
     |                                                   ^^^^^^^^^^^^^^^^^ cannot infer type
     | 
    ::: /home/dabreegster/.cargo/registry/src/github.com-1ecc6299db9ec823/structopt-0.3.21/src/lib.rs:1114:35
     |
1114 |         I::Item: Into<OsString> + Clone,
     |                                   ----- required by this bound in `structopt::StructOpt::from_iter`
     |
     = note: cannot satisfy `_: Into<OsString>`
help: consider specifying the type argument in the function call
     |
192  |         let gj = clockboard(Point::new(0.0, 0.0), Params::from_iter::<I>(Vec::new()));
     |                                                                    ^^^^^

```

This is saying that the compiler can't figure out the type of the thing inside the `Vec::new()` -- because it's an empty vector, kind of hard to guess what's inside it. Usually Rust would see that we're passing the empty vector into a function that expects a `Vec<Something>`, but for https://docs.rs/structopt/0.3.21/structopt/trait.StructOpt.html#method.from_iter, it can take different types in the vector. Anything that satisfies all of those traits / constraints listed. You don't need to understand too much about that at this point -- it'll make more sense when you need to write a generic method -- but in short, we can give a hint as it says and just say we're handing in a `String`:

```
let args: Vec<String> = Vec::new();
Params::from_iter(args)
```